src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -133,21 +144,73 @@ const Comments = ({
 
         <ul className={styles.commentsContainer}>
           {orderBy(comments, ['created'], ['desc']).map(comment => {
-            const { createdBy } = comment;
+            const { createdBy, resolvedBy } = comment;
 
-            return (
+            return comment.resolved ? (
+              <li className="govuk-body-s">
+                <p className="govuk-!-margin-0">
+                  <strong>Comment resolved</strong>
+                </p>
+                <p className="govuk-!-margin-0">
+                  <FormattedDate format="dd/MM/yy HH:mm">
+                    {comment.resolved}
+                  </FormattedDate>
+                </p>
+                <p className="govuk-!-margin-0">
+                  {`by ${resolvedBy?.firstName} ${resolvedBy?.lastName}`}
+                </p>
+                <Details className="govuk-!-margin-0" summary="See comment">
+                  <p className="govuk-!-margin-0">
+                    <strong>
+                      {`${createdBy.firstName} ${createdBy.lastName}`}
+                    </strong>
+                  </p>
+                  <p className="govuk-!-margin-0">
+                    {'Created: '}
+                    <FormattedDate format="dd/MM/yy HH:mm">
+                      {comment.created}
+                    </FormattedDate>
+                  </p>
+                  {comment.updated && (
+                    <p className="govuk-!-margin-0">
+                      {'Updated: '}
+                      <FormattedDate format="dd/MM/yy HH:mm">
+                        {comment.updated}
+                      </FormattedDate>
+                    </p>
+                  )}
+                  <p className="govuk-!-margin-top-3">{comment.content}</p>
+                </Details>
+                <ButtonText
+                  onClick={() => {
+                    resolveComment(comment.id, false);
+                  }}
+                >
+                  Unresolve
```

Comment:
Do we not need to show and hide this button based on permissions?

Reply:
Seems I've incorrectly assumed this was wrapped `canComment`, but it is not. But I tested this by creating a user that only had viewing permission, and they cannot see comments at all. And all users that can edit a release should also have permission to resolve comments. So it doesn't appear to be an issue, but I'll make the Unresolve button conditionally appear based on `canComment` such that it doesn't accidentally get revealed if permissions change in the future.